frontend/src/citizen-frontend/income-statements/IncomeStatements.tsx

```diff
@@ -118,6 +120,8 @@ export default React.memo(function IncomeStatements() {
     [page]
   )
 
+  const [children, fetchChildren] = useApiState(() => getChildren(), [])
+
```

Comment:
Would it make sense to fetch the income statements of all children with one request, instead of first fetching the childre nand then fetching their income statements?

With slow connection it looks like this: 

https://user-images.githubusercontent.com/70927/154414979-5e61aa86-2ba6-499e-8d43-337d4d9744bb.mov

This is just an idea, not sure how hard this would be to implement.